extensions/wikia/AdEngine/AdEngine2ContextService.class.php

```diff
@@ -24,6 +24,11 @@ public function getContext( Title $title, $skinName ) {
 				$sevenOneMediaCombinedUrl = ResourceLoader::makeCustomURL( $wg->Out, ['wikia.ext.adengine.sevenonemedia'], 'scripts' );
 			}
 
+			$monetizationServiceAds = null;
+			if ( !empty( $wg->AdDriverUseMonetizationService ) && !empty( $wg->EnableMonetizationModuleExt ) ) {
+				$monetizationServiceAds = F::app()->sendRequest( 'MonetizationModule', 'index' )->getData()['data'];
```

Comment:
I'm going to nitpick a little bit. I hope you'll forgive me :japanese_ogre: 

What happens when there is error or empty response from Monetization service?